2015/proposals/akshay.md

```diff
@@ -0,0 +1,107 @@
+# Title
+
+Add taxonomic name resolution to the EcoData Retriever to facilitate data 
```

Comment:
Since this is your title you should use

~~~
# Add taxonomic name resolution to the EcoData Retriever to facilitate data science approaches to ecology
~~~

Sorry if it wasn't clear from the template.